source/api/discovery/0.1/index.md

```diff
@@ -0,0 +1,609 @@
+
+The most effective information to add beyond the basic resource list is the timestamp at which the resource was last modified (including the initial modification that created it). If we know these dates, we can add them to the Activities and order the list such that the most recent activities occur last. The time at which the document update process finished is given in the `endTime` property. Consumers will then process the list of Activities in reverse order, from last to first, stopping when they encounter an Activity they have already processed in a previous run. 
+
+Example Level 1 Activity:
+
+```
+{
+  "type": "Update",
+  "object": {
+    "id": "https://example.org/iiif/1/manifest",
+    "type": "Manifest"
+  },
+  "endTime": "2017-09-20T00:00:00Z"
+}
+```
+
+#### 2.1.3. Level 2: Complete Change List
+
+At the most detailed level, a log of all of the Activities that have taken place can be recorded, with the likelihood of multiple Activities per IIIF resource.  This allows the additional types of "Create" and "Delete", enabling a synchronization process to remove resources as well as add or update them. The list might end up very long if there are many changes to resources, however this is not a typical situation, and the cost is still reasonable as each entry is short and can be compressed both on disk and at the HTTP(S) transport layer.
+
+Example Level 2 Activity:
+
+```
+{
+  "type": "Create",
+  "object": {
+    "id": "https://example.org/iiif/1/manifest",
+    "type": "Manifest"
+  },
+  "endTime": "2017-09-20T00:00:00Z"
+}
+```
+
+### 2.2. Pages of Changes
+
+Activities are collected together into pages that together make up the entire set of changes that the publishing system has made.  Pages reference the previous and next pages in that set, and the overall collection that they are part of.  The Activities are then listed in time order.
+
+
+```
+{
+  "id": "https://example.org/activity/page-1",
+  "type": "OrderedCollectionPage",
+  "partOf": {
+    "id": "https://example.org/activity/top-collection",
+    "type": "OrderedCollection"
+  },
+  "prev": {
+    "id": "https://example.org/activity/page-0",
+    "type": "OrderedCollectionPage"
+  },
+  "next": {
+    "id": "https://example.org/activity/page-2",
+    "type": "OrderedCollectionPage"
+  },
+  "orderedItems": [
+    {
+      "type": "Update",
+      "object": {
+        "id": "https://example.org/iiif/9/manifest",
+        "type": "Manifest"
+      },
+      "endTime": "2018-03-10T10:00:00Z"
+    },
+    {
+      "type": "Update",
+      "object": {
+        "id": "https://example.org/iiif/2/manifest",
+        "type": "Manifest"
+      },
+      "endTime": "2018-03-11T16:30:00Z"
+    }
+  ]
+}
+```
+
+### 2.5. Collections of Pages
+
+As the number of Activities is likely too many to usefully be represented in a single Page, they are collected together into a Collection as the initial entry point. The Collection references the URIs of the first and last pages.
```

Comment:
> be represented in a single Page

capital P on page?

Reply:
Shorthand for Ordered Collection Page. There also isn't a Collection, and no type in the data that is an Activity.  So it's all a bit fuzzier than the Prezi set of classes.